src/controller/CHIPDeviceController.h

```diff
@@ -698,7 +698,8 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
 private:
     DevicePairingDelegate * mPairingDelegate;
 
-    CommissioneeDeviceProxy * mDeviceBeingCommissioned = nullptr;
+    DeviceProxy * mDeviceBeingCommissioned           = nullptr;
+    CommissioneeDeviceProxy * mDeviceInProcessOfPase = nullptr;
```

Comment:
Maybe
```suggestiona
    CommissioneeDeviceProxy * mDeviceInPASEEstablishment = nullptr;
```
but I don't feel super-strongly about it...

Reply:
Changed now in most recent push.